.github/workflows/liblab-generate-sdks.yaml

```diff
@@ -14,10 +14,20 @@ jobs:
   build-and-pr:
     name: Generate SDKs and create PRs
     runs-on: ubuntu-latest
+    if: github.run_number != 1
     env:
       LIBLAB_TOKEN: ${{ secrets.LIBLAB_TOKEN }}
       GITHUB_TOKEN: ${{ secrets.LIBLAB_GITHUB_TOKEN }}
     steps:
+      - name: Check if secrets are set
+        run: |
+          if [[ -n "${{ secrets.LIBLAB_TOKEN }}" && -n "${{ secrets.LIBLAB_GITHUB_TOKEN }}" ]]; then
+            echo "Secrets are set, proceeding with the workflow"
+          else
+            echo "Error: Secrets LIBLAB_TOKEN and LIBLAB_GITHUB_TOKEN are required, see https://developers.liblab.com/tutorials/integrate-with-github-actions/#create-tokens"
```

Comment:
Do we need both `echo`s?

Reply:
It will add the same output in the failed step, I think it's helpful, it can't hurt.